src/librbd/librbd.cc

```diff
@@ -4124,6 +4139,28 @@ extern "C" int rbd_aio_open_by_id_read_only(rados_ioctx_t p, const char *id,
   return 0;
 }
 
+extern "C" int rbd_features_to_string(rados_ioctx_t io, uint64_t features,
+                                      char *str_features, size_t *size)
+{
+  std::string get_str_features = librbd::rbd_features_to_string(features, nullptr);
+  uint64_t expected_size = get_str_features.size();
+  if (*size < expected_size) {
```

Comment:
```suggestion
  if (*size <= expected_size) {
```

... or preferably, just set `expected_size = get_str_features.size() + 1`